pdm-pep517 changed name to pdm-backend with minor changes.
author: micketeer@gmail.com

--- pyproject.toml.orig 2024-03-01 19:33:52.181523013 +0100
+++ pyproject.toml      2024-03-01 19:45:38.960430755 +0100
@@ -60,6 +60,6 @@
 
 [build-system]
 requires = [
-    "pdm-pep517>=1.0.0",
+    "pdm-backend",
 ]
-build-backend = "pdm.pep517.api"
+build-backend = "pdm.backend"

